providers/k8s/resources/discovery.go

```diff
@@ -1170,13 +1170,13 @@ func assetName(ns, name string) string {
 	return ns + "/" + name
 }
 
-func OwnerReferencesFilter(refs []metav1.OwnerReference, filter map[string]struct{}) bool {
+func OwnerReferencesFilter(refs []metav1.OwnerReference, filter ...string) bool {
```

Comment:
nit: can those be private funcs?

Reply:
we want to re-use them in the mondoo-operator as it needs to do the same filtering